docs/reset-psw-user.rst

```diff
@@ -0,0 +1,24 @@
+====================================
+Reset specific user password by shell
+====================================
+
+
+---------------
+In case you don't remember the user password it is easy to do change by CLI. For example:
+
+- Run the shell mode in Django:
+
+   ``docker-compose run --rm web ./manage.py shell``
+
+- To create multiple model objects at once (eg. 10) use the ``create_batch()``
```

Comment:
Do we need this paragraph as it doesn't use `create_batch` below?

Reply:
ops dammit c&p